lua_protobuf/generator.py

```diff
@@ -183,10 +186,10 @@ def package_function_prefix(package):
 def message_function_prefix(package, message):
     return '%s%s_' % (package_function_prefix(package), message)
 
-def message_open_function_name(package, message):
+def message_open_function_name(package, message, namespace=[]):
```

Comment:
Using `[]` as a default value is one of the most common Python gotchas. See http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments for more.

The correct way to write this is:

```
def func(namespace=None):
    namespace = namespace or []
```
or
```
def func(namespace=None):
    if namespace is None:
        namespace = []
```